ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -26,6 +26,14 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/openshift-kni/numaresources-operator/bin/exporter
   to: resource-topology-exporter
+- dockerfile_path: Dockerfile.openshift.tests
+  from: os-min8
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift-kni/numaresources-operator/bin/.
```

Comment:
is the` /.` at the end the right format ?

Reply:
If the CI passed, I think it is